modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,65 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-If you are deploying applications to xref:index.adoc[Runtime Manager] or APIs on xref:api-manager::index.adoc[API Manager] and your company's security policies require that you run penetration testing on them, there is a way for you to run these. Given that such tests can interfere with other tenants, penetration testing is only allowed on your own workers. It is not allowed for any other services in the Anypoint Platform.
+If your company security policies require that you perform vulnerability and penetration testing 
+before deploying to Anypoint Platform, you must request a Security Assessment before proceeding 
+with your testing on Anypoint Platform.
 
-External vulnerability scans, via an Approved Scanning Vendor (ASV), as described in section 11.2.2 of the PCI standard may be performed at will and do not require prior approval. Penetration testing as described by section 11.3.1 of the PCI standard does require prior approval from MuleSoft.
+Because these tests can interfere with other tenants, MuleSoft allows penetration testing on only your workers
+but not for other Anypoint Platform services.
 
-CloudHub Workers, by default, do not use static IPs and cannot be properly tested as their IPs may change. Please enable static IPs, as described in xref:deploying-to-cloudhub.adoc#static-ips[Static IPs].
+== Prerequisites
+
+Before penetration testing, enable static IPs for the app.
+
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change.
+For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
+
+== Request a Security Assessment 
+
+To request a Security Assessment, follow the steps in https://help.salesforce.com/articleView?id=000336416&type=1&mode=1[Salesforce Security vulnerability assessment and penetration test]:
+
+. https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form]. 
+. Assign a security resource at your company to review and validate your findings.
++
+[IMPORTANT]
+You must complete this step before following up with Salesforce. 
++
+Use the following documents to identify common false positives or security issues related to settings:
```

Comment:
Is this tied to the notice; is this what your security resource should be doing, or something separate that you would do in addition to what that resource does? Also before contacting SF?

Reply:
Please doublecheck that these statements, whether bulleted or not, are chronological.

To request an assessment >
complete the form >

assign internal resource to review the form? >

use these documents (to test your form findings? AFTER showing to internal resource?) >

send your form to SF (so, all the prev actions are supposed to precede following up with SF, not just assigning internal resource?) >

send separate vulnerability findings (so...are there TWO tests, one filling out the SF form and another using docs to test?) and include all those details? >

oh, and read up on how to send these findings besides that detailed list? >

and don't bother if you have a custom development process? >

and schedule an assessment?

This does not sound precisely chrono to me.
Seems we should start off saying you can't request a SF assessment if you have customized dev.

Then, if you qualify, follow the steps in that doc OR follow the steps here?

1. Get the form
2. Complete the form (does this involve testing or just environment statistics?)
3. Have an assigned rep check the form before sending to SF
x. ? Perform separate testing after completing and checking the form?
x. ?Send form one place, test results another?
last. Request an assessment.

Reply:
Went back to source and revised with corrected process.